src/plays/digital-delight/components/CategoryFact.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import Math from '../components/images/math.png';
+import Year from '../components/images/year.png';
+import Date from '../components/images/date.png';
+import './Categoryfact.css';
+
+function CategoryFact() {
+  const [fact, setFact] = useState('Click on buttons below to get facts by category');
+  const [selectedVoice, setSelectedVoice] = useState('Zira');
+  const [showSelecter, setShowSelecter] = useState(false);
+  const speechHandler = (msg) => {
+    // This for loop helps to give the selected voice
+    for (let i = 0; i < window.speechSynthesis.getVoices().length; i++) {
+      if (window.speechSynthesis.getVoices()[i].name === selectedVoice) {
+        msg.voice = window.speechSynthesis.getVoices()[i];
+      }
+    }
+    window.speechSynthesis.cancel();
+    window.speechSynthesis.speak(msg);
+  };
+
+  const getFact = async (type) => {
+    const options = {
+      method: 'GET',
+      url: `https://numbersapi.p.rapidapi.com/random/${type}`,
+      params: { fragment: 'true', json: 'true' },
+      headers: {
+        'X-RapidAPI-Key': process.env.REACT_APP_DIGITSDELIGHT_APIKEY,
+        'X-RapidAPI-Host': 'numbersapi.p.rapidapi.com'
+      }
+    };
+    try {
+      const response = await axios.request(options);
+      setFact(response.data.number + ' is ' + response.data.text);
+      speechHandler(
+        new SpeechSynthesisUtterance(response.data.number + ' is ' + response.data.text)
+      );
+      setShowSelecter(true);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  const handleVoiceChange = (event) => {
+    setSelectedVoice(event.target.value);
+  };
+
+  return (
+    <>
+      <div className="digits-delight-voice-selection-container" style={{ marginBottom: '-25px' }}>
```

Comment:
@Shivam-Katare, I would not recommend the negative margins, see if another way is possible.